frog/imports/api/sessions.js

```diff
@@ -42,14 +42,9 @@ Meteor.methods({
       }).fetch();
       let nextNum = 1;
       if (prev && prev.length > 0) {
-        prev.sort((x, y) => y.slug - x.slug);
```

Comment:
Here you sort alphabetically, that must have caused problems with "10" being less that "2"